report.tex

```diff
@@ -43,7 +43,7 @@
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \begin{abstract}
-
+The main objective of this report is to document the investment of $R~100~000$ in a share in a sector in the Johannesburg Stock Exchange (JSE). The \textit{Consumer goods and services sector} was chosen on the basis of its stability and growth over the past year. The growth was confirmed by analysing the $2017$ budget review which shows that the sector accounted for $70~\%$ of the economic growth in $2016$. A forecast that this sector will experience a growth of up to $2~\%$ in the next year was also used to confirm the stability of the sector. Within the chosen sector, the British American Tobacco (BAT) was the chosen share as it tops the JSE top 40 index and shows a more stable trend, in comparison to its closest rivals in the sector, in the closing prices analysed over the past year. A quartely analysis reveals that the BAT inmproved in governance of the past year due to the appointment of a key board member. Merger deals on future products also increased their presence in those markets. The company strategy to further multiply their presence in these markets was also used to predict good long term value in the initial investment.
```

Comment:
Perhaps it would be more descriptive to state at the outset that the report documents an investment into the BAT.
Perhaps we should name the 'key board member'. 
Perhaps we should name the closest rivals.
Perhaps we should mentions the merger deal and the new markets.